Guides/Anomalies/Anomalies & You - Anomaly Types

```diff
@@ -0,0 +1,50 @@
+[bold]Containment Tip:[/bold] Treat them a lot like the Pyroclastic Anom', though spacing the tile undearneath is optional but recommended, these can be contained with reinforced glass but once over 50-60% severity, use Reinforced Plasma Glass Windows and check on the windows damage regularly.  Wear thermal protection when interacting with ice anomalies, AND BRING A WELDING TOOL.
+
+[head=3]Electric Anomalies[/head]
+[bold]Properties:[/bold] Generate high-voltage discharges, creating electrical surges that can damage electronics and injure personnel. They can short out power grids in their vicinity.
+[bold]Containment Tip:[/bold] Ground the containment area using a grounding rod. These anomalies are best contained in shielded areas away from critical electronics. Relocating the Anom with a Gauntlet is an option as well.
+
+[head=3]Gravitational Anomalies[/head]
+[bold]Properties:[/bold] Warp gravitational fields, causing objects and people to be pulled toward or pushed away from them. They can make moving around difficult and sometimes create dangerous high-gravity zones. Be warned, radiation is also emitted from these anom's.
+[bold]Containment Tip:[/bold] Wear a radiation suit, use lockers or other dragable objects like your A.P.E. to nudge the Anomaly into a better position, I would suggest using Plasma Windows (small/thin) to contain the anom and stop the radiation from causing harm.
+
+[head=3]Flesh Anomalies[/head]
+[bold]Properties:[/bold] Composed of living, organic material, flesh anomalies may spawn hostile creatures if left unchecked. They also emit a foul odor and can spread, affecting their surroundings.
+[bold]Containment Tip:[/bold] Quarantine flesh anomalies and have weapons ready in case of hostile entity emergence. Consider immediate disposal if containment is compromised.
+
+[head=3]Bluespace Anomalies[/head]
+[bold]Properties:[/bold] Bluespace anomalies can warp space, leading to localized teleportation effects. They may cause items or personnel to randomly teleport or phase in and out of the area.
+[bold]Containment Tip:[/bold] Avoid placing unanchored valuable or sensitive equipment nearby, as it may be affected by teleportation. Try to Avoid keeping this anom near the edges of the station and/or public hall ways. Use the APE to stabilize or neutralize quickly.
+
+[head=3]Tech Anomalies[/head]
+[bold]Properties:[/bold] Interact with and sometimes enhance electronic devices in unpredictable ways, potentially overloading nearby machinery or toggling it's functions.
+[bold]Containment Tip:[/bold] Keep an eye on your APE, as this Anom can and will turn it off or change the particle type it will shoot. Only keep non-essential equipment near Tech anomalies, as they may short-circuit or alter devices function, such as bolting/opening/toggling power.
```

Comment:
Minor tweak
```suggestion
[bold]Containment Tip:[/bold] Keep an eye on your APE, as this Anom can and will turn it off or change the particle type it will shoot. Only keep non-essential equipment near Tech anomalies, as they may short-circuit or alter device's functions, such as bolting/opening/toggling power.
```